lua/bufferline/animate.lua

```diff
@@ -2,38 +2,18 @@
 -- animate.lua
 --
 
-local float2nr = vim.fn.float2nr
-local reltime = vim.fn.reltime
-local reltimefloat = vim.fn.reltimefloat
```

Comment:
I actually don't like this change, for the same reasons as the `vim.api.` changes.